lib/ts/ink_code.h

```diff
@@ -28,6 +28,7 @@
 #include "ts/ink_defs.h"
 #include <openssl/md5.h>
```

Comment:
Maybe we can move this line into the `ifndef`.

Reply:
Done.